src/commands/blocks.ts

```
@@ -147,6 +151,8 @@ export const blocksCommand = define({
 	},
 	toKebab: true,
 	async run(ctx) {
+		// Initialize i18n with CLI language argument
+		i18n.initialize(ctx.values.lang);
```

ESLint

[error] Unsafe argument of type `any` assigned to a parameter of type `string | undefined`.

(ts/no-unsafe-argument)